apps/wallet-ui/.env.example

```diff
@@ -0,0 +1,12 @@
+# Root domain for the app
+NEXT_PUBLIC_ROOT_DOMAIN=ecosystem.localhost:3000
+# Thirdweb client ID
+NEXT_PUBLIC_THIRDWEB_CLIENT_ID=
+# Thirdweb API key
+THIRDWEB_SECRET_KEY=
+# SimpleHash API key
+SIMPLEHASH_API_KEY=
+# Whether or not to include testnets in the UI
+NEXT_PUBLIC_INCLUDE_TESTNETS=1
```

Comment:
how come this is an env var? I would've expected this to be potentially a setting on the ecosystem but not an env var, or is it because you need a way to test but then it's always mainnet for prod?

Reply:
I need a way to test but don't want testnets to be displayed in prod. In prod you can just exclude this var and it defaults to false